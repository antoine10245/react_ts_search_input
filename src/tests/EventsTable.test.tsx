import * as React from "react";
import { shallow } from "enzyme";
import EventsTable from "../components/EventsTable.component";

describe("EventsTable component", () => {
  const subject = shallow(<EventsTable events={[]} />);

  it("renders without crashing", () => {
    expect(subject).toMatchSnapshot();
  });
});
