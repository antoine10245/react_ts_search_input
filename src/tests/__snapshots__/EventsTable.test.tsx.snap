// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventsTable component renders without crashing 1`] = `
<styled.div
  className="col-sm-12 col-xl-8"
>
  <h2
    className="alert-light text-center"
  >
    no events found
  </h2>
</styled.div>
`;
